### ISSUE TITLE
java.nio.BufferOverflowException

### ISSUE THREAD

**Issue by WolfgangFahl** [albfan/jmeld#47]
java.nio.BufferOverflowException
	at java.nio.CharBuffer.put(CharBuffer.java:922)
	at java.nio.CharBuffer.put(CharBuffer.java:950)
	at org.jmeld.diff.JMDiff.filter(JMDiff.java:249)
	at org.jmeld.diff.JMDiff.diff(JMDiff.java:110)
	at org.jmeld.util.node.JMDiffNode.diff(JMDiffNode.java:294)
	at org.jmeld.ui.FileComparison.doInBackground(FileComparison.java:94)
	at org.jmeld.ui.FileComparison.doInBackground(FileComparison.java:17)

**Comment by WolfgangFahl** [comment]
Please find below the fix by increasing the buffer size and which is more important making it configurable ...
 
```
public class JMDiff
 {
+  public static int BUFFER_SIZE=100000;
   // Class variables:
   // Allocate a charBuffer once for performance. The charbuffer is used to
   //   store a 'line' without it's ignored characters. 
-  static final private CharBuffer inputLine = CharBuffer.allocate(10000);
-  static final private CharBuffer outputLine = CharBuffer.allocate(10000);
```

**Comment by WolfgangFahl** [comment]
Would you please release a new version to maven to make this fix available ...

**Comment by albfan** [comment]
Thanks for working on these. This bug is really a stopper to use jmeld.

Can you provide a PR so I can merge?

**Comment by albfan** [comment]
Sure, no problem

**Comment by albfan** [comment]
Fixed on 44cd1ca
